src/components/AppMap.vue

```diff
@@ -139,6 +139,14 @@
           <AppMapFilterMainButton v-for="(v, type) in markerComponents" :key="type" :type="type" :label="v.filterLabel" :icon="v.filterIcon" @toggle="updateMarkers" />
         </div>
         <b-checkbox switch v-model="showKorokIDs" @change="updateKorokIDs">Show Korok IDs</b-checkbox>
+        <b-checkbox v-model="korokTypeOpen"  switch >Korok Types</b-checkbox>
```

Comment:
I think the UX is somewhat confusing. It's in the filter pane so I would have expected this feature to *filter* visible Korok markers, not add/remove extra search groups.